content/changes.md

```diff
@@ -8,11 +8,29 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## April 30, 2024 
+
+### Improvements 
+
+- **Allow candidates to move forward with partial solutions in certain Project Code Challenges**
+
+  For Project Code Challenges that enforce a limited number of submissions, candidates with incomplete solutions are no longer forced to use their remaining submission attempts in order to proceed with the assessment.
+
+### Challenges 
+
+- **Create Drupal quiz** 
```

Comment:
```suggestion
- **Drupal Quiz** 
```

The ticket is to create the quiz, but the new challenges list should just state the name of the challenge, "Drupal Quiz" in this case.